src/model/read_xyz.cu

```diff
@@ -548,8 +620,14 @@ void initialize_position(
   }
 
   std::vector<std::string> atom_symbols;
-  auto filename_potential = get_filename_potential();
+  auto filename_potential = get_out_potential();
```

Comment:
This line should be separately writen in the #ifdef and #else parts

Reply:
> This line should be separately writen in the #ifdef and #else parts

OK, I've fixed it and it compiles.